### PR TITLE
Admin controller/v2 api

### PR DESCRIPTION
**Pull request by royanik17**

### CODE REVIEW

**Comment by ayanit84** [comment]
Fix the "out-of-date" branch

**Comment by ayanit84** [comment]
Make sure to use "Squash and merge" when merging
image

**Review by ayanit84 — CHANGES_REQUESTED** [review]
need changes

**Comment by royanik17** [comment]
OK

**Comment by royanik17** [comment]
H

File: src/main/java/com/experimental/product/community/membershipservice/sevice/MemberAdminServiceV2.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                    existingMember.getUnit(),
+                    existingMember.getJoiningDate(),
+                    existingMember.getMarried(),
+                    existingMember.getActive(),
+                    existingMember.getPaymentOptions(),
+                    existingMember.getPreferences(),
+                    existingMember.getFamily(),
+                    existingMember.getAuditData()));
+            if (updateMemberRequest.getPaymentOptions() != null && !updateMemberRequest.getPaymentOptions().isEmpty()) {
+                updatedMember = updatedMember.updatePaymentOptions(updateMemberRequest.getPaymentOptions());
```
Comment (ayanit84) [inline src/main/java/com/experimental/product/community/membershipservice/sevice/MemberAdminServiceV2.java:52]: what exactly it is doing? referring _ updatedMember_ and then again assigning to the same variable _ updatedMember_

**Review by ayanit84 — CHANGES_REQUESTED** [review]
please check

**Review by ayanit84 — APPROVED** [review]

**Pull request merged**